src/styles/styles.js

```diff
@@ -2171,12 +2171,22 @@ function getZoomCursorStyle(isZoomed, isDragging) {
 
 /**
  * @param {Boolean} isZoomed
+ * @param {Number} imgWidth
+ * @param {Number} imgHeight
  * @return {Object}
  */
-function getZoomSizingStyle(isZoomed) {
+function getZoomSizingStyle(isZoomed, imgWidth, imgHeight) {
+    const MIN_ZOOM_SIZE = 2000;
```

Comment:
I would probably move this `const` into https://github.com/Expensify/App/blob/main/src/CONST.js, similar to what we do for another constant in this file: https://github.com/Expensify/App/blob/e171217e4d9cf089d0d7f4053b92c42107fe312e/src/styles/styles.js#L2357

Reply:
Ok.

Reply:
Fixed